operations/helm/charts/mimir-distributed/templates/admin-api/admin-api-dep.yaml

```diff
@@ -82,6 +82,9 @@ spec:
             {{- toYaml .Values.admin_api.resources | nindent 12 }}
           securityContext:
```

Comment:
I think it would be best if we allow the user to override the whole `securityContext` instead of just the privileged field. So by default it would be `readOnlyRootFilesystem: true`, but users can replace the entire block with whatever they need. What do you think?

Reply:
If we go that way, I would also put an empty `securityContext` in the values.yaml with a docstring describing what it does.

Reply:
@Logiraptor  I was thinking the same, I believe `readOnlyRootFilesystem: true` is a best practice.  We can iterate on this and have it defined in the values.yaml with additional security best practices ?

Reply:
I think we should use the nginx gateway as an example and make this similar. https://github.com/grafana/mimir/blob/613839578f5550e9eb5a586168f90089d3ba2d60/operations/helm/charts/mimir-distributed/values.yaml#L1227